src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { PropsWithChildren } from 'react';
+import * as S from './Stepper.style';
+
+export type StepperProps = PropsWithChildren<any>;
+
+const Stepper: React.FC<StepperProps> = ({ children, ...rest }: StepperProps) => (
+	<S.Stepper {...rest}>
+		{children &&
+			React.Children.map(children, (child, index) => [
+				<div className="stepper__progress-bar" />,
```

Comment:
Does it mean you render a `.stepper__progress-bar` next to each step?

Reply:
yep